persistence/sql/migrations/sql/20230920171028000000_identity_search_index.cockroach.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS identity_credential_identifiers_nid_identifier_gin ON identity_credential_identifiers USING GIN (nid, identifier gin_trgm_ops);
```

Comment:
To use [online schema changes](https://www.cockroachlabs.com/docs/v23.1/online-schema-changes) with cockroach, we recommend to manually copy this index definition and run it before applying migrations.